stdimage/models.py

```diff
@@ -144,6 +144,10 @@ def delete_variations(self):
             variation_name = self.get_variation_name(self.name, variation)
             self.storage.delete(variation_name)
 
+    def __getstate__(self):
+        attributes = self.__dict__.copy()
```

Comment:
This will include files streams, this is why the super method overwrites get state. You'll need to come up with a slightly more sophisticated implementation that only copies the variations (without which in turn should also not include file streams).